components/AdminList/AdminList.js

```diff
@@ -0,0 +1,135 @@
+import { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+import s from "./AdminList.module.scss";
+import Tick from "../../public/svg/tickIcon.svg";
+import QuestionMark from "../../public/svg/questionMarkIcon.svg";
+import Dagger from "../../public/svg/daggerIcon.svg";
+import DeleteIcon from "../../public/svg/deleteIcon.svg";
+import EditIcon from "../../public/svg/editIcon.svg";
+import AddAdminForm from "../AddAdminForm";
+import EditAdminForm from "../EditAdminForm";
+
+export default function AdminList() {
+  const data = [
+    {
+      id: 0,
+      email: "zelykrostyslav@email.com",
+      status: "loggedIn",
+    },
+    {
+      id: 1,
+      email: "denysgolovko@email.com",
+      status: "loggedIn",
+    },
+    {
+      id: 2,
+      email: "lebronjames@email.com",
+      status: "notLoggedIn",
+    },
+    {
+      id: 3,
+      email: "giannisantetokounmpo@email.com",
+      status: "deleted",
+    },
+  ];
+
+  const LOGGED = "loggedIn";
+  const NOTLOGGED = "notLoggedIn";
+  const DELETED = "deleted";
```

Comment:
В таких случаях можно создать подобие типа `enum`
```suggestion
const eAdminStatus = {
  LOGGED: 'logged',
  NOTLOGGED: 'notLoggedIn',
  DELETED: 'deleted',
}
```